### DISCUSSION TITLE
CUE as a configuration language for a streaming platform

### DISCUSSION THREAD

**Discussion by cueckoo** [cue-lang/cue#1072] [Show and tell]
_Originally opened by @owulveryck in https://github.com/cuelang/cue/discussions/1072_

Hi,

I am working on a data mesh initiative. As an architect on the project, I want to prove the value of a streaming platform as part of my job.
The streaming platform is, according to me, not only an infrastructure component but needs some features such as data validation.
As a demo, I coded something with CUE and the Go SDK.
It is documented here and holds a couple of references to gists that expose the code with some tests.

https://blog.octo.com/en/pov-a-streaming-communication-platform-for-the-data-mesh

Any comment welcome.

**Comment by myitcv** [comment]
Thanks for sharing this, @owulveryck!
